### ISSUE TITLE
Confusing doc prometheus-operator vs kube-prometheus

### ISSUE THREAD

**Issue by nicgirault** [prometheus-operator/prometheus-operator#2619]
I am a noob who try to setup some monitoring for my cluster & apps. I lost 2 days of work trying to use kube-prometheus because of these lines: https://github.com/coreos/prometheus-operator/blob/b602243f57c988f9a2fc7efbedcbfad9ce611bcf/README.md#prometheus-operator-vs-kube-prometheus

I was really upset that kube-prometheus ask me to install some strange dependencies (jsonnet) with awful config files. Why there is no helm chart for this awesome package?? I saw multiple times the chart prometheus-operator but as I read the doc linked above I was sure I didn't want to use prometheus-operator. At last I realized that prometheus-operator chart was packaging kube-prometheus stack but it took me around 10 hours playing around to realize this.

I would highly suggest to rename the helm chart `kube-prometheus` or at least add a note in the doc above explaining that prometheus-operator helm chart deploys the kube-prometheus stack.

Thanks

**Comment by brancz** [comment]
We've suggested this repeatedly to the helm charts maintainers (we don't maintain the charts, they're entirely community maintained). @vsliouniaev this is not the first time we're getting this report, could you please consider renaming the helm chart? Thanks! :) 

> I was really upset that kube-prometheus ask me to install some strange dependencies (jsonnet) with awful config files.

Nobody is forcing you to use this, we feel these tools are solving our problem at hand very well, and there are many happy users of this jsonnet based stack. If you could tell us what you felt was difficult there, we're more than happy to add more documentation! :)

**Comment by vsliouniaev** [comment]
In my opinion, renaming the chart will just cause a bunch of chaos in the short term, then will be confusing in terms of feature-parity with the jsonnet version. There is already a `stable/prometheus` chart, which doesn't use the operator, so having a `stable/prometheus-operator` chart that will be marked as deprecated and moving it to something like `stable/kube-prometheus` will be ambiguous as to the relationship of that to everything else.

Is adding documentation about the existence of the chart and its general intention enough to solve this problem?

**Comment by metalmatze** [comment]
What about having `stable/kube-prometheus` chart for a period of time (6 months plus) that is an exact copy of the current `stable/prometheus-operator` chart for people to move over to `stable/kube-prometheus` before deprecating the current one?!

We can probably add as much documentation as we want and some people will still not read it (been there myself so no blaming). I strongly advocate for renaming the chart.

**Comment by vsliouniaev** [comment]
That's an interesting suggestion - do you know if it's feasible to do this in the helm/charts project with all the hoops that need to be jumped through?

**Comment by metalmatze** [comment]
I have no clue about anything related to helm/charts :wink: Was just thinking how to move forward.

**Comment by vsliouniaev** [comment]
The suggestion when I raised this was wait a bit to see if the documentation change has a positive effect first. There was also a suggestion to just host the chart externally, which would probably cause similar issues again.

**Comment by Eeemil** [comment]
To add more confusion, I'm about to do an upgrade of the old coreos/prometheus-operator chart from 0.0.29. The [old prometheus-operator docs](https://github.com/coreos/prometheus-operator/tree/release-0.29/helm) seem to require to install both coreos/prometheus-operator and coreos/kube-prometheus charts.

Google led me here. After further investigation, I found [more information regarding](https://github.com/coreos/prometheus-operator/issues/592#issuecomment-436778117) the prometheus-operator migration. It seems like the chart `stable/prometheus-operator` includes what `coreos/kube-prometheus` contained, and nowadays, `kube-prometheus` contains `prometheus-operator`, but `kube-prometheus` isn't a helm chart anymore :sweat_smile:

**Comment by LiliC** [comment]
@Eeemil Sorry we don't actually support that Helm chart anymore as we do not actually make use of Helm, so like it says in the docs:
https://github.com/coreos/prometheus-operator/tree/master/helm 

> Issues and pull requests should be tracked using the helm/charts repository.

But maybe ask on the prometheus-operator k8s slack channel, some users there use that chart and often help other helm users. :)

**Comment by Eeemil** [comment]
Yeah, sorry, I wasn't really asking for help setting up the helm chart, just adding my thoughts regarding confusion of the concepts "prometheus-operator" and "kube-prometheus". I googled "prometheus-operator vs kube-prometheus" to gather more information and it lead me here. It would perhaps be nice to explain the differences of the concepts either in some readme, perhaps under "Prometheus Operator vs. kube-prometheus vs. community helm chart" in the main readme of this repo or under `helm/README.md`.

**Comment by paulfantom** [comment]
It is already described in main README file in this repository: https://github.com/coreos/prometheus-operator#prometheus-operator-vs-kube-prometheus-vs-community-helm-chart ?
